rules/kubernetes/policies/general/allowing_to_update_a_malicious_pod.rego

```diff
@@ -20,9 +20,9 @@ package builtin.kubernetes.KSV048
 import data.lib.kubernetes
 import data.lib.utils
 
-workloads := ["deployments", "daemonsets", "statefulsets", "replicationcontrollers", "replicasets", "jobs", "cronjobs"]
+workloads := ["pods", "deployments", "jobs", "cronjobs", "statefulsets", "daemonsets", "replicasets", "replicationcontrollers"]
 
-changeVerbs := ["update", "create", "*"]
+changeVerbs := ["create", "update", "patch", "delete", "deletecollection", "impersonate", "*"]
```

Comment:
wondering why we are using asterisk (*) here - wildcard character that matches any string. We have added more verbs now. Do we still need * ?

Reply:
If we are using the wildcard (*), then there is no need to define verbs explicitly, since the asterisk(*) will match any verb.

Reply:
I concur that including a wildcard (*) in the verbs list negates the need to define verbs explicitly, as the asterisk (*) will match any verb. The asterisk (*) was already present in the verbs list, and I introduced additional verbs to ensure alignment between the oss and commercial.